### PR TITLE
Completed css-layout-classes Exercise - Bingle Padilla

### PR DESCRIPTION
**Pull request by codewithbingle**
css-layout classes

### CODE REVIEW

File: css-layout-classes/styles.css [hunk 1]
```diff
@@ -0,0 +1,28 @@
+body {
```
Comment (thebearingedge) [inline css-layout-classes/styles.css:1]: If you add a rule set like this, your padding will not add to the width of your elements, and the text should fall in the right place.

```css
* {
  box-sizing: border-box;
}
```

Comment (thebearingedge) [reply css-layout-classes/styles.css:1]: https://developer.mozilla.org/en-US/docs/Web/CSS/box-sizing

File: css-layout-classes/index.html [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+      <div class="column-one-third">
+        <img src="./images/link.png">
+      </div>
+      <div class="column-one-third">
+        <h2>Super Smash Bros. Ultimate</h2>
+        <p>Super Smash Bros. Ultimate received universal acclaim from both critics and players, with some critics calling it the best installment in the series. It received praise for its large amount of content and fine-tuning of existing Smash gameplay elements, although its online mode was widely criticized.</p>
+      </div>
+      <div class="column-one-third">
+        <img src="./images/yoshi.png">
+      </div>
```
Comment (thebearingedge) [inline css-layout-classes/index.html:27]: You'll want to add a css class to style the column-one-third divs.

```css
.column-one-third {
  width: calc(100% / 3);
}
```

**Review by thebearingedge — CHANGES_REQUESTED** [review]

**Comment by codewithbingle** [comment]
css-layout classes-take2

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**